### DISCUSSION TITLE
How to get the color "By Block"？

### DISCUSSION THREAD

**Discussion by liunx7594** [mozman/ezdxf#417] [Q&A]
![image](https://user-images.githubusercontent.com/55911351/116491530-ab8ef680-a8cc-11eb-823f-306aea6f0ff0.png)

I already know "true_color "and" dxf.color "and the color of the layer.
And the index number of "dxf.color" of "By Block" is 1

How to get the color "By Block"？

**Comment by mozman** [comment]
This code shows how the drawing add-on resolves the entity color:

https://github.com/mozman/ezdxf/blob/3af906028911dff4c0e322b9b39c2fd1d742f909/src/ezdxf/addons/drawing/properties.py#L458 

If you write your own DXF render engine you should study the drawing add-on!

**Comment by liunx7594** [comment]
Great code, is there any way to copy color conveniently?

**Comment by liunx7594** [comment]
Or I want to just copy line segments or text
